### PR TITLE
VZ-3946.  Update WKO to v3.3.3

### PR DESCRIPTION
**Pull request by michael-cico**
# Description

Update to WKO v3.3.3 with support for Istio 1.10+
- Update helm chart
- Update BOM entry
- Add chart override to set `istioLocalhostBindingsEnabled: false` for Istio 1.10+ support on by default

Fixes VZ-3946

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

**Review by markxnelson — APPROVED** [review]

File: application-operator/apis/weblogic/v8/domain_status.go [hunk 1]
```diff
@@ -26,6 +26,9 @@ type DomainStatus struct {
 	// A brief CamelCase message indicating details about why the domain is in this state
 	Reason string `json:"reason,omitempty"`
 
+	// Unique id of the last introspector job that was processed for this domain.
+	LastIntrospectJobProcessedUID int `json:"lastIntrospectJobProcessedUid,omitempty"`
```
Comment (mbenmclean) [inline application-operator/apis/weblogic/v8/domain_status.go:30]: Was there documentation in the release about this having to go in?

**Review by mbenmclean — APPROVED** [review]

File: application-operator/apis/weblogic/v8/domain_status.go (hunk 1 above)
Comment (michael-cico) [reply application-operator/apis/weblogic/v8/domain_status.go:30]: It's a status field value.  It's weird, this shows up as a diff in git when I compared CRD versions between 3.3.2 and 3.3.3, but I can't find this field when I grep it.  Going to back that out.

Comment (michael-cico) [reply application-operator/apis/weblogic/v8/domain_status.go:30]: Checking with Lenny.  I can't grep it in the CRD yaml, but it is documented here

https://github.com/oracle/weblogic-kubernetes-operator/blob/main/documentation/domains/Domain.md

Comment (michael-cico) [reply application-operator/apis/weblogic/v8/domain_status.go:30]: Checked with Lenny, it's legit.

**Pull request merged**